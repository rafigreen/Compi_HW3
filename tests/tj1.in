//rule 18.1

override bool f(){
    return true;
}
override int f(){
    return 0;
}

void main(){
    bool a = true and f();
}