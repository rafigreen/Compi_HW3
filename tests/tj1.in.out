---end scope---
---end scope---
line 11: ambiguous call to overloaded function f
